tests/Client/TestLoader.php

```diff
@@ -33,7 +28,7 @@ class TestRepo implements RepoFileFetcherInterface
      *
      * @var array[]
      */
-    public array $fetchMetadataArguments = [];
+    public array $maxBytes = [];
```

Comment:
The doc for this property needs to be updated

Reply:
Fixed.